crates/bevy_input/src/input.rs

```diff
@@ -29,6 +29,35 @@ use bevy_ecs::schedule::State;
 /// * Using [`Input::clear_just_pressed`] or [`Input::clear_just_released`] instead.
 /// * Calling [`Input::clear`] or [`Input::reset`] immediately after the state change.
 ///
+/// ## Performance
+///
+/// For all operations, the following conventions are used:
+/// - **n** is the number of stored inputs.
+/// - **m** is the number of input types.
```

Comment:
I'm not sure of the meaning of "input types", there's only one type `T`

Reply:
```suggestion
/// - **m** is the number of input parameters passed to the method.
```